src/SprykerEco/Yves/Heidelpay/Handler/HeidelpayEasyCreditHandler.php

```diff
@@ -0,0 +1,151 @@
+<?php
+
+/**
+ * MIT License
+ * For full license information, please view the LICENSE file that was distributed with this source code.
+ */
+
+namespace SprykerEco\Yves\Heidelpay\Handler;
+
+use ArrayObject;
+use Generated\Shared\Transfer\ExpenseTransfer;
+use Generated\Shared\Transfer\PaymentTransfer;
+use Generated\Shared\Transfer\QuoteTransfer;
+use SprykerEco\Shared\Heidelpay\HeidelpayConfig;
+use SprykerEco\Yves\Heidelpay\Dependency\Client\HeidelpayToCalculationClientInterface;
+use SprykerEco\Yves\Heidelpay\Dependency\Client\HeidelpayToPriceClient;
+use SprykerEco\Yves\Heidelpay\Dependency\Client\HeidelpayToQuoteClientInterface;
+
+class HeidelpayEasyCreditHandler extends HeidelpayHandler
+{
+    public const PAYMENT_PROVIDER = HeidelpayConfig::PROVIDER_NAME;
```

Comment:
If you use these constants only in this class make them protected otherwise move them into Config model.